### DISCUSSION TITLE
O Curso App: API com ASP.NET Core, CQRS e EF Core

### DISCUSSION THREAD

**Discussion by marcionevesdev** [balta-io/7196#7] [Perguntas e Respostas] [answered]
Ola Balta. 
  O Curso App  API com ASP.NET Core, CQRS e EF Core diz na descricao que : 

"Este curso é a construção da API para uma solução completa (Backend, Frontend e Mobile) de um aplicativo de Lista de Tarefas, implementando as melhores soluções na arquitetura da API."

No entanto,  não encontrei nada de Frontend e Mobile.

**Comment by andrebaltieri** [comment, marked as answer]
@marcioLegiaodeAco https://balta.io/cursos/criando-um-todo-app-com-aspnet-core-firebase-angular-flutter
